etc/sysconfig/pacemaker.in

```diff
@@ -212,6 +212,15 @@
 #
 # Default: PCMK_remote_port="3121"
 
+# PCMK_remote_pid1 (Advanced Use Only)
+#
+# Specify behavior when the Pacemaker Remote process is not running as PID 1.
+# * If set to "full", Pacemaker Remote behaves as though it's running as PID 1.
+# * If set to "vars", Pacemaker Remote loads environment variables from
+#   /etc/pacemaker/pcmk-init.env, while all other PID 1 behavior is disabled.
```

Comment:
* If unset, Pacemaker Remote performs neither behavior when it is not PID 1.
* If set to "full", Pacemaker Remote performs both PID 1 behaviors even when it is not PID 1.
* If set to any other value (such as "vars"), Pacemaker Remote loads environment variables from /etc/pacemaker/pcmk-init.env even when it is not PID 1, but does not read dead subprocesses.